lib/clearance/password_strategies/bcrypt_migration_from_sha1.rb

```diff
@@ -22,10 +29,12 @@ def initialize(user)
       end
 
       def authenticated?(password)
+        warn "#{Kernel.caller.first}: #{DEPRECATION_MESSAGE}"
```

Comment:
Would it be a good idea to extract all of these warnings to some kind of `warn` utility method which would take care of prepending the message with the caller?